### ISSUE TITLE
money supply number must be wrong

### ISSUE THREAD

**Issue by brenzi** [encointer/explorer#5]
the side view shows me
money supply 490616
for currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992

If I query the storage directly, I get 
```
0x2ebf164a5bb618ec6caad31488161b237e24d75efa3040286767b620d9183989
BalanceEntry: 275'621'727'887'989'712'908'770'490'616
```
and after more ceremonies
```
BalanceEntry: 275'625'998'580'708'170'386'706'587'816
```

the population of this currency are the following 20 accounts:
```
May 20 14:00:56 gesell bash[30993]: NCTR balance for 5EUXHVefrGT18zqRJJ8vxJTRrmbgA6UUUrAQ3QkmLMLPf7Hd is 759.6467827664018218539 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5Ff3YLGxAtSqpAmorA9srq3zMGFhBuJiQgDvKeXP4ph5yzqp is 759.6467827664018218539 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5CtryXjK4U6VSTyqZEZ44cBUMcwca7uvNKv49CsXcAyaXseQ is 759.6467827664018218539 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5Ci4bxeWR8GAaaWxmksYdcn55tdrRs26waABLVwTYdPxQhkY is 758.6471216386167508327 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5CtadXZuibXHaoNP9VKsoGRc2ih2tHvzdMrDG2GSXFuN89yG is 759.6467827664018218539 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5HK4piM876WwSsBQma1KxVqaAyNX1YVStBNdYwaidtussWyU is 758.6709598486188923913 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5ELedUmuWEoa4gRrrK47e8rTz58b6XumpAZXqNWq686enUaw is 744.07966627646991403754 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5HNQEKg19fN7zH3NzEDjGi8L2mKyCdXHAeXpg1ghXxjDKWT8 is 744.0172693478465030947 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5DtnTQa2GHmtUtQmUj9jB2FUzkJnPEM2VWAqVstoZiKA6hxj is 742.22182128078611764273 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5GpowyZe7tn8Zk4nxc2E94CHEqyZMoK2hJqUiCEEeyTGB8Nu is 744.3161813316299226947 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5FhikZdSV8vP99ikPDoREcj4LMiZ4ex7apmAgBJgaqg6VANG is 746.13493654242033942676 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5G1aNEGP6GtxonXk2RjgadbHxQuzsnfjJKNAfiFk5bMEiQt1 is 737.4347839033422062965 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5GQgvv1CwqCqWpQL9uvyd9ADRhX4bPEos9w4VEKVgtM4bYb7 is 743.2666072275195674891 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5HMso2QkkkdymvdwGTUugz5QEiWRUZzqbutHDWgZMtMXdF7Z is 749.0958576196559179289 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5H9SsunoVnHiBz72FsYySdKtxDhUvPGHMiwbHZzuBoaSmHWd is 739.28758136876928198423 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5EHXpwi4T929taFmtWygzKCMwGU4G37vYyEiNcYKwZtsAVem is 740.365328195698828365 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5EnVxV3GBvn1qvXMouQYiq623YaH4PEm59XFftytcRKZ9tWL is 748.0713791935538846455 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5DwNewuRFxwHYztZ1RPv9JokNjAz8grHcCPzTMgSE89tAJhB is 742.31799576111588755356 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5HYBJtbQbBJ9Bwx7QGx8Dqr2oWH2Bm4c6ez9wBeTAK2rGnSK is 739.41564913094233701593 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
May 20 14:00:57 gesell bash[30993]: NCTR balance for 5E4etyd3aJh934MpUC1sLaQgtRy13hqZbU7spKujucSR7Sj8 is 745.26650321226152466104 in currency 49UkeKNfbLGgZk5h64TrEnirAc27hbdGyDwSPFLg6992
```

total issuance should be the sum of these, so roughtly 20*750 = 15000

It seems like the moneyissuance is not correctly parsed, because
```
octave:29> b = 275625998580708170386706587816;
octave:34> b/2^(64+20)
ans =    1.4250e+04
octave:35> 20*750
ans =  15000
```

**Comment by brenzi** [comment]
this must be caused by a type error fixed in 3f9ae5fd256d87ac5b41914d028c728b0a0e55ce

**Comment by demyanrogozhin** [comment]
This was fixed in #6, as total supply is also Float in BigInt disguise ;).
